server/s3_object_metadata.h

```diff
@@ -257,7 +266,14 @@ class S3ObjectMetadata : private S3ObjectMetadataCopyable {
   std::string get_acl_as_xml();
   unsigned int get_number_of_parts() const { return this->obj_parts; }
   unsigned int get_number_of_fragments() const { return this->obj_fragments; }
+  void set_number_of_parts(unsigned int parts) { this->obj_parts = parts; }
```

Comment:
can we make this and others as inline functions !!

Reply:
ok. will make them inline